modules/price/src/InvalidArgumentException.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace Drupal\commerce_price;
+
+class InvalidArgumentException extends \InvalidArgumentException {
```

Comment:
We can remove this one. It doesn't add anything over the builtin \InvalidArgumentException.